feature/widget/message-list/src/main/kotlin/app/k9mail/feature/widget/message/list/BaseMessageListWidgetProvider.kt

```diff
@@ -13,9 +13,8 @@ import com.fsck.k9.activity.MessageList.Companion.intentDisplaySearch
 import com.fsck.k9.search.SearchAccount.Companion.createUnifiedInboxAccount
 import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
-import com.fsck.k9.ui.R as UiR
 
-open class MessageListWidgetProvider : AppWidgetProvider(), KoinComponent {
+open class BaseMessageListWidgetProvider : AppWidgetProvider(), KoinComponent {
```

Comment:
We don't want anyone to create an instance of this directly. So this could be an abstract class.